### DISCUSSION TITLE
How to change the language used to post to discord?

### DISCUSSION THREAD

**Discussion by netchild** [aussig/BGS-Tally#257] [Q&A]
Hi,

where can I find the setting to change the language? I want to have posts to discord in english, not in the language of my desktop. The release notes don't tell how to switch and in the config template, and the bgs tally window I see not setting for the language. There's also no plugin-settings tap in the edmc settings window.

While it is nice that there are translations, I would like to have the option to stay with English also in the EDMC windows. I play ED in English as I can't stand the English translations and are in an English speaking ED discord, and it would be nice if I don't need to switch languages in my head when I look at EDMC.

Bye,
Alexander.

**Comment by netchild** [comment]
Ok, seems there shall be a settings page, so ignore this. I will open a bug report that no settings page is available.

**Comment by aussig** [comment]
You’ve asked two different questions here. 

If you want the **whole** of EDMC in English then that’s in the general EDMC settings (File -> Settings -> Configuration). BGS-Tally will automatically pick up on that language too. 

If you want just **BGS-Tally discord posts** to be in English (as per the title of this issue), then go to BGS-Tally settings and select English in the drop-down there.
